Bio/motifs/__init__.py

```diff
@@ -24,16 +24,18 @@ def create(instances, alphabet=None):
 
 
 def parse(handle, format):
-    """Parse an output file from a motif finding program.
+    """
+    Parse an output file from a motif finding program.
 
     Currently supported formats (case is ignored):
-     - AlignAce:      AlignAce output file format
-     - MEME:          MEME output file motif
-     - MAST:          MAST output file motif
-     - TRANSFAC:      TRANSFAC database file format
-     - pfm:           JASPAR-style position-frequency matrix
-     - jaspar:        JASPAR-style multiple PFM format
-     - sites:         JASPAR-style sites file
+    - AlignAce:      AlignAce output file format
+    - MEME:          MEME output file motif
+    - MINIMAL:       MINIMAL MEME output file motif
+    - MAST:          MAST output file motif
+    - TRANSFAC:      TRANSFAC database file format
+    - pfm:           JASPAR-style position-frequency matrix
+    - jaspar:        JASPAR-style multiple PFM format
+    - sites:         JASPAR-style sites file
```

Comment:
You've not yet fixed the indentation - this is breaking the bullet point RST rendering, at least as rendered on rst.nija.org which is an easy to use online renderer.